blogging/admin.py

```diff
@@ -1,7 +1,23 @@
 from django.contrib import admin
-
-# Register your models here.
 from blogging.models import Post, Category
 
-admin.site.register(Post)
-admin.site.register(Category)
\ No newline at end of file
+
+class CategoryInPostInline(admin.TabularInline):
+    model = Category.posts.through
+    extra = 2
+
+
+class PostAdmin(admin.ModelAdmin):
+    inlines = [
+        CategoryInPostInline,
+    ]
+
+
+class CategoryAdmin(admin.ModelAdmin):
+    inlines = [
```

Comment:
You don't need inlines in the category admin, you're just removing posts from it, the exclude statement does that.